inc/settings.php

```diff
@@ -24,47 +24,257 @@ function siteorigin_corp_settings_init() {
+					'description' => esc_html__( 'Display related posts on the single post page.', 'siteorigin-corp' )
+				),						
+				'ajax_comments' => array(
+					'type' => 'checkbox',
+					'label' => esc_html__( 'Ajax Comments', 'siteorigin-corp' ),
+					'description' => esc_html__( 'Keep the conversation flowing with ajax loading comments.', 'siteorigin-corp' ),
+					'teaser' => true,
+				),
+			),											
+		),
+
+		// Sidebar.
+		'sidebar' => array(
+			'title' => esc_html__( 'Sidebar', 'siteorigin-corp' ),
+			'fields' => array(			
+				'position'	=> array(
+					'type'	=> 'select',
+					'label'	=> esc_html__( 'Position', 'siteorigin-corp' ),
+					'description' => esc_html__( 'Choose the sidebar position.', 'siteorigin-corp' ),
+					'options' => array(
+						'right' => esc_html__( 'Right', 'siteorigin-corp' ),
+						'left'  => esc_html__( 'Left', 'siteorigin-corp' ),
+					),
+				),
+				'width' => array(
+					'label'       => esc_html__( 'Width', 'siteorigin-unwind' ),
+					'type'        => 'measurement',
+					'live'        => true,
+				),										
+			),		
+		),		
+
 		// Footer.
 		'footer' => array(
 			'title' => esc_html__( 'Footer', 'siteorigin-corp' ),
 			'fields' => array(
-
 				'text' => array(
 					'type' => 'text',
 					'label' => esc_html__( 'Footer Text', 'siteorigin-corp' ),
 					'description' => esc_html__( "{sitename} and {year} are your site's name and current year.", 'siteorigin-corp' ),
 					'sanitize_callback' => 'wp_kses_post',
 				),
+				'attribution' => array(
+					'type' => 'checkbox',
+					'label' => esc_html__( 'Hide SiteOrigin Attribution', 'siteorigin-north' ),
+					'description' => esc_html__( 'Hide the SiteOrigin link in your footer.', 'siteorigin-north' ),
+					'teaser' => true,
+				),
 			),
-		),		
-
+		),
 	) ) );
 }
 add_action( 'siteorigin_settings_init', 'siteorigin_corp_settings_init' );
 
 /**
+ * Add custom CSS for the theme settings
+ *
+ * @param $css
+ *
+ * @return string
+ */
+function siteorigin_corp_settings_custom_css( $css ) {
+	// Custom CSS.
+	$css .= '/* style */
+	.widget-area {
+	width: ${sidebar_width};
+	}';
+	return $css;
+}
+add_filter( 'siteorigin_settings_custom_css', 'siteorigin_corp_settings_custom_css' );
+
+if ( ! function_exists( 'siteorigin_corp_menu_breakpoint_css' ) ) :
+/**
+ * Add CSS for mobile menu breakpoint.
+ */
+function siteorigin_corp_menu_breakpoint_css( $css, $settings ) {
+	$breakpoint = isset( $settings[ 'theme_settings_navigation_mobile_menu_collapse' ] ) ? $settings[ 'theme_settings_navigation_mobile_menu_collapse' ] : 768;
+
+	$css .= '@media (max-width: ' . intval( $breakpoint ) . 'px) {
+		#masthead .search-toggle {
+			margin-right: 20px;
+		}
+
+		#masthead #mobile-menu-button {
+			display: inline-block;
+		}
+
+		#masthead .main-navigation ul {
+			display: none;
+		}
+
+		#masthead .main-navigation .search-icon {
+			display: none;
+		}
+	}
+	@media (min-width: ' . ( 1 + $breakpoint ) . 'px) {
```

Comment:
Is there a reason why we don't run intval() here, but we do for the other media query?